packages/local-relay/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@fitbit/local-relay",
+  "version": "1.8.0-pre.0",
+  "description": "Local implementation of the Developer Relay",
+  "author": "Fitbit, Inc.",
+  "homepage": "https://github.com/Fitbit/developer-bridge/tree/main/packages/local-relay#readme",
+  "license": "BSD-3-Clause",
+  "main": "lib/local-relay.js",
```

Comment:
How does this file get built? I don't see a matching file in sources.

Reply:
`https://github.com/Fitbit/developer-bridge/pull/182/commits/fa15c8c6999dae5376d7440d49487249ae424bc6`